### PR TITLE
feat: adds new integration banner

### PR DESCRIPTION
**Pull request by mendesmcmg**
# Description

- Adds new integration banner on causes page

## Does this pull request close any open issues?

- [#prod3-174](https://ribon.atlassian.net/browse/PROD3-174?atlOrigin=eyJpIjoiMjE2ZjY5YzI2M2NlNDM1NmFhYjA4NzcxN2QyN2YzYmIiLCJwIjoiaiJ9)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Dangerous changes
- [ ] Chore (change that does not effect how the code works, eg: change color)
- [ ] Tests

## How to test

- open via integration

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,10 +9,12 @@ npm-debug.*
 *.mobileprovision
 *.orig.*
 web-build/
+.tool-versions
```
Comment (thaisribon) [inline .gitignore:12]: just checkin, is this supposed to be here?

File: src/components/moleculars/Banner/index.tsx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 /* eslint-disable react-native-a11y/has-valid-accessibility-ignores-invert-colors */
 import { theme } from "@ribon.io/shared/styles";
+import React from "react";
```
Comment (thaisribon) [inline src/components/moleculars/Banner/index.tsx:3]: just checkin, is this supposed to be here?

File: src/components/moleculars/UserSupportBanner/index.tsx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import { useTranslation } from "react-i18next";
 
 import { logEvent } from "services/analytics";
-
+import React from "react";
```
Comment (thaisribon) [inline src/components/moleculars/UserSupportBanner/index.tsx:4]: just checkin, is this supposed to be here?

File: tsconfig.json [hunk 4]
```diff
@@ -4,7 +4,8 @@
     "strict": true,
     "baseUrl": "src",
     "sourceMap": true,
-    "sourceRoot": "/"
+    "sourceRoot": "/",
```
Comment (thaisribon) [inline tsconfig.json:7]: just checkin, is this supposed to be here?

File: src/components/moleculars/BannerInfo/styles.ts [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    display: "flex",
+    flexDirection: "column",
+    width: "80%",
+  },
+  title: {},
+  subTitle: {
+    marginBottom: theme.spacingNative(4),
+  },
+  text: {
+    ...defaultBodySmRegular,
```
Comment (thaisribon) [inline src/components/moleculars/BannerInfo/styles.ts:30]: i think this should be: defaultBodySmMedium, right?

File: src/components/moleculars/Banner/index.tsx (hunk 2 above)
Comment (mendesmcmg) [reply src/components/moleculars/Banner/index.tsx:3]: oh sorry, I was having errors without it before, but solved it and forgot to remove. thanks for the catch!

File: src/components/moleculars/BannerInfo/styles.ts (hunk 5 above)
Comment (mendesmcmg) [reply src/components/moleculars/BannerInfo/styles.ts:30]: you're right! don't know what happened here

**Review by thaisribon — APPROVED** [review]

**Pull request merged**